src/TextField.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module TextField
+ */
+class TextField extends React.Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			value: props.value,
+			error: props.error
+		};
```

Comment:
you only need something in `this.state` if you are going to be dynamically updating it independently of `props` passed in by the parent. Currently, errors are _always_ provided by the parent, but if you develop this to add something like a char counter that would have to show an error while someone is typing, it would be good to add `error` to `this.state` at that point. Probably better _not_ to have it in `this.state` until you need it for `this.setState()`